src/components/ShopCategory/ShopCategory.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import './ShopCategory.scss';
+import { Link } from 'react-router-dom';
+import { sectionsLinks } from '../../helpers';
+
+interface Props {
+  cards: Good[];
+}
+
+export const ShopCategory: React.FC<Props> = ({ cards }) => {
+  const imageWidth = 368;
+  const imageGap = 16;
+  const imageCount = 3;
+
+  return (
+    <div
+      className="Category"
+      style={{
+        width: `${imageWidth * imageCount + (imageGap * (imageCount - 1))}px`,
+      }}
+    >
+      <div className="Category__Title">
+        <p className="Category__Title-text">
+          Shop by category
+        </p>
```

Comment:
Это же заголовок? Тогда H2 вместо P. 
```suggestion
<h2 className="Category__Title">
  Shop by category
</h2>
```

Reply:
+